Makefile

```diff
@@ -273,6 +276,13 @@ print-config:
 	@echo LIBS = $(LIBS)
 	@echo PLATFORM = $(PLATFORM)
 
+# These definitions are needed to get the right __declspec() annotation on
+# Windows.  See htslib/hts_defs.h
+$(LIBHTS_OBJS) : BUILDING_LIBRARY = -DHTS_BUILDING_LIBRARY
+$(LIBHTS_OBJS:.o=.pico) : BUILDING_LIBRARY = -DHTS_BUILDING_LIBRARY
+$(PLUGIN_OBJS) : BUILDING_LIBRARY = -DHTS_BUILDING_LIBRARY
+$(PLUGIN_OBJS:.o=.pico) : BUILDING_LIBRARY = -DHTS_BUILDING_LIBRARY
```

Comment:
iMHO the low-tech approach of putting `#define HTS_BUILDING_LIBRARY` at the top of all the relevant _*.c_ files would be better than cluttering the Makefile and build logs by having this explicit on the command line.